gm4_soul_glass/data/gm4/advancements/soul_glass.json

```diff
@@ -7,7 +7,7 @@
     "title": {
       "translate": "%1$s%3427655$s",
       "with": [
-        "Corruption at its finest",
+        "Corruption at Its Finest",

```

Comment:
The capitalization on this advancement is a bit odd, should it be:
Corruption at its Finest
Corruption At Its Finest

I think it looks odd to not capitalize the "at", but don't know if either of these two are any better either.

Reply:
yup, I derped with the 'its'. I was following a guide but I realized that other vanilla advancements don't capitalize possessive adjectives.